plugins/node/plugin.zsh

```diff
@@ -0,0 +1,21 @@
+# Color definitions
+GEOMETRY_COLOR_NODE_NPM_VERSION=${GEOMETRY_COLOR_NODE_NPM_VERSION:-black}
+
+# Symbol definitions
+GEOMETRY_SYMBOL_NODE_NPM_VERSION=${GEOMETRY_SYMBOL_NODE_NPM_VERSION:-"⬡"}
+GEOMETRY_NODE_NPM_VERSION=$(prompt_geometry_colorize $GEOMETRY_COLOR_NODE_NPM_VERSION $GEOMETRY_SYMBOL_NODE_NPM_VERSION) 
+
+
+geometry_prompt_node_setup(){} 
+
+geometry_prompt_node_render(){
+    if (( $+commands ['node'] == 0 )); then
```

Comment:
I am working on the detecting the yarn.lock file and .nvmrc file but I am running into problems. This is the line I want to add:
`if [[-e yarn.lock]] || [[ -f .nmvrc && -r .nvrmc && -n "$(head -1 .nvmrc)]]; then"`.
I want add this above the 
`if (( $+commands[node] == 0)); then`

I tested it using a simple .nvmrc file in a directory jsproj using this steps 
![screenshot from 2017-01-23 22 58 29](https://cloud.githubusercontent.com/assets/15849318/22234701/b82296dc-e1bf-11e6-8ff8-276956856d01.png)